code/__HELPERS/hallucinations.dm

```diff
@@ -111,7 +123,7 @@ GLOBAL_LIST_EMPTY(all_ongoing_hallucinations)
 			to_chat(nearby_living, pick(optional_messages))
 
 /// Global weighted list of all hallucinations that can show up randomly.
-GLOBAL_LIST_INIT(random_hallucination_weighted_list, generate_hallucination_weighted_list())
+GLOBAL_LIST_INIT_TYPED(random_hallucination_weighted_list, /list, generate_hallucination_weighted_list())
```

Comment:
brother what the fuck are you on

Reply:
does that even work???

Reply:
Yes

Reply:
![image](https://github.com/user-attachments/assets/c3d9c34e-a6e7-4441-ae33-290e30b4939d)

Reply:
thanks I hate it